bonus_guides/D_mix_tasks.md

```diff
@@ -373,6 +376,16 @@ Add the channel to your `web/channels/user_socket.ex` handler, for example:
     channel "rooms:lobby", HelloPhoenix.RoomChannel
 ```
 
+#### `mix phoenix.gen.presence`
+
+This task will generate a Presence tracker for your application. You can pass the module name as an optional argument,
```

Comment:
Could we change the narritive so that it reads like the rest of the descriptions?

Something like:

 > +This task will generate a Presence tracker. The module name can be passed as an argument, `Presence` is used if no module name is passed.